tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/chanqWin.tss

```diff
@@ -0,0 +1,35 @@
+timeseries scalar settingsfile: /perm/mo/mocm/prg/oslisflood/ecmwf/lisflood/tests/data/LF_ETRS89_UseCase/settings/inflow.xml_c228be65-d717-4449-b7bb-efeb1411327b.xml date: Wed Feb 17 16:12:40 2021
```

Comment:
We can delete the folder "tests/data/LF_ETRS89_UseCase/**reference_mergeMinLoops12**" before the push